py/tests/test_processing.py

```diff
@@ -128,6 +128,18 @@ def test_normalize_user_agent(must_normalize):
         assert "contexts" not in event
 
 
+def test_validate_pii_selector_spec():
+    sentry_relay.validate_pii_selector_spec("test")
+    sentry_relay.validate_pii_selector_spec("$user.id")
+    sentry_relay.validate_pii_selector_spec("extra.'sys.argv'.**")
+
+    with pytest.raises(ValueError):
```

Comment:
Does the error come with an error message? If yes: is it actionable, or does it help identifying why the selector is not valid?

Reply:
The error messages are good, I've added some assertions to the python tests